public/dorequest.php

```diff
@@ -277,6 +285,7 @@ function DoRequestError(string $error, ?int $status = 200, ?string $code = null)
         $response['AchievementID'] = $achIDToAward;
         break;
 
+    // TODO deprecate
```

Comment:
The DLL still uses this to show friend activity in the clients. It's buried in the overlay. I don't think I've ever used it. We could probably deprecate it.